applications/CSharpWrapperApplication/CSharpWrapperApplication.py

```diff
@@ -0,0 +1,10 @@
+from KratosCSharpWrapperApplication import *
+application = KratosCSharpWrapperApplication()
+application_name = "KratosCSharpWrapperApplication"
+application_folder = "CSharpWrapperApplication"
+
+# The following lines are common for all applications
+from .. import application_importer
+import inspect
+caller = inspect.stack()[1] # Information about the file that imported this, to check for unexpected imports
+application_importer.ImportApplication(application,application_name,application_folder,caller,__path__)
```

Comment:
please use the new mechanism, this will otherwise stop working again soon

Reply:
Of course

Reply:
:+1: